### ISSUE TITLE
RC1 Checklist

### ISSUE THREAD

**Issue by albertlast** [SimpleMachines/SMF#4485]
Since the rc1 https://github.com/SimpleMachines/SMF2.1/milestone/6 is hard to follow 
if a point is covered or not
i like to create this check list.

"Error Sources/Security.php" is not clear to me if it fixed or not.

legend:
No mark
![grafik](https://user-images.githubusercontent.com/1782906/47770595-43053580-dce0-11e8-8dc5-97cef83ce390.png)

mark
![grafik](https://user-images.githubusercontent.com/1782906/47770614-56180580-dce0-11e8-80cd-3134ffa30359.png)

strike
~~text~~ = pr is merged

at smf team feel free to edit by your self

**Comment by Gwenwyfar** [comment]
Can we add this to RC as well? #1401 

Much more important than the avatars issue IMO.

**Comment by albertlast** [comment]
ask @colinschoen for me is this list only impoirtant to see if they covered by a pr or  not

**Comment by Sesquipedalian** [comment]
> Can we add this to RC as well? #1401 
> Much more important than the avatars issue IMO.

Agreed. Added.

**Comment by Sesquipedalian** [comment]
I've added #1178 to RC1 as well. We don't want to put out a release candidate that contains a way for people to bypass part of the permissions system.

**Comment by sbulen** [comment]
This one always bothered me, & felt like it should be resolved before RC: Issue #2939 .   force_ssl = 1 (login & reg only) doesn't work at all.  It should either work, or the option should be eliminated.

**Comment by Sesquipedalian** [comment]
Added.

**Comment by sbulen** [comment]
I have a suggestion...   

Before we officially tag the release as RC, can we have a few days' warning?   I have a normal barrage of tests I run that I'd like to do once we think we're ready.   

When Beta 3 came out, it felt to me like we had a flurry of activity, & then it was immediately released.  But it had a few kinda big issues, we could have easily caught.   I'd like to avoid that if we could for RC.  

A few days of 'test only', then we could doc & prioritize bugs.   Allow for possible regressions due to all of the updates.

**Comment by albertlast** [comment]
My pov is that the talk is out of scope of this issue.
Create a own issue or create a topic with this questions.

**Comment by illori** [comment]
@sbulen that should be discussed on team boards ideally. usually a release has to be scheduled/approved etc. so there should be notice posted there.

**Comment by frandominguezl** [comment]
@colinschoen Seems like this is almost ready to go! Maybe you can have a look at the remaining PRs to get RC1 ready to release?

**Comment by albertlast** [comment]
Update the list to get a better overview again

**Comment by SychO9** [comment]
close this :octocat: ?
